src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -218,6 +221,34 @@ ScrollDown.propTypes = {
 
 export default function Banner() {
   const classes = useStyles();
+  const { telescopeUrl } = useSiteMetadata();
+  const [sha, setSha] = useState('');
+  const [version, setVersion] = useState('');
+  const [githubUrl, setGitHubUrl] = useState('');
```

Comment:
Don't bother with this now, but you could use a single state Object vs. splitting this out into three.  It's fine for state to be a nested object in react.